### DISCUSSION TITLE
[Bug] XCH sent to an unfamiliar wallet

### DISCUSSION THREAD

**Discussion by vazo222** [Chia-Network/chia-blockchain#10876] [General]
### What happened?

3.2 chia disappeared from my wallet. in the old version of chia, no operations were shown, and in the new one, incomprehensible transactions appeared.
xch1sq68hqkthcpjrz0eddeesum9pg2chkpxhgygh7g45mdplkz6c66s8sx4ma - the address of the wallet to which 3.2 chia was transferred for some reason

### Version

1.2.11-1.3.0

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by andriitalkstech** [comment]
Very similar issue
https://www.reddit.com/r/chia/comments/tc1wr6/xch_was_stolen_from_hundreds_of_people/
